src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -167,9 +182,21 @@ export class Inquiries extends React.Component<Props, State> {
     alert("Sorry, there was an error with your submission, please try again")
   }
 
+  trackSuccessfulSubmission() {
+    let ids: string[] = []
+    ids = this.state.selected_conversations.map(value => value.id)
```

Comment:
Why init with an empty array first? And I’d expect TS to figure out `ids`’s type with:

```
const ids = this.state.selected_conversations.map(value => value.id)
```